packages/server-wallet/src/wallet/index.ts

```diff
@@ -240,6 +273,19 @@ export class Wallet implements WalletInterface {
     );
     return {outbox: mergeOutgoing(outgoing.map(n => n.notice)), channelResult};
   }
+
+  async joinChannels(channelIds: ChannelId[]): MultipleChannelResult {
```

Comment:
Do we need `joinChannel` if we have a `joinChannels` that already works for a single element `channelIds` array?

Reply:
There's a question of maintaining backwards compatibility. But I think that the server-wallet's API is closer to the One True API than the `client-api-schema`'s opinion.

Reply:
I've removed `joinChannel`, `createChannel` and `updateChannelFunding` in 
1684ff4

Reply:
The backwards compatibility struggle is real. Especially when `createChannel` would guarantee you a `SingleChannelResult`, but `createChannels` doesn't. I suggest that we put these methods back in - it costs very little to maintain them, and we reduce the hassle burden for users of the wallet.

Reply:
For reference, this is [the diff](https://github.com/statechannels/the-graph/pull/180/files#diff-290c3918607a033549623003e44452b1) for the payment manager:

![Screen Shot 2020-10-05 at 3 38 45 PM](https://user-images.githubusercontent.com/29596/95093723-fb378600-0720-11eb-8892-c39c5fbf7fea.png)